src/webpack/loaders/javascript.js

```diff
@@ -13,29 +13,20 @@ export default {
     ))
 
     return {
-      babel: {
-        babelrc: path.join(projectPath, '.babelrc'),
-        plugins: babelPlugins(action, coverage)
-      },
-
       plugins: action === actions.DEVELOP ? [new HotModuleReplacementPlugin()] : [],
 
-      resolve: {
-        extensions: fileExtensions.list.JAVASCRIPT
-      },
-
       module: {
-        loaders: [
+        rules: [
           {
             test: fileExtensions.test.JAVASCRIPT,
             include: [
               path.join(projectPath, 'src'),
               ...userPaths
             ],
-            loader: 'babel',
-            query: {
-              // enabling it breaks source maps
-              compact: false
+            loader: 'babel-loader',
+            options: {
```

Comment:
Sure, it's a better name than `query`, but I wonder if this means that the loaders no longer take URL like "query params"